### PR TITLE
Fix: Add tests for `ConfigResolver`

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] adds tests for the `ConfigResolver`

Related to #247.

### CODE REVIEW

File: tests/Config/ConfigResolverTest.php [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+    public function provideDefaultConfigFileNameAndAvailableDefaultConfigFileNames(): array
+    {
+        return [
+            [
+                '.twig_cs',
+                [
+                    '.twigcs',
+                    '.twigcs.dist',
+                ],
+            ],
```
Comment (localheinz) [inline tests/Config/ConfigResolverTest.php:152]: @Ciloe @OwlyCode @yguedidi

In the context of #247, we could add more data sets here to ensure that configuration files are resolved in the specified order - what do you think?

File: tests/Config/ConfigResolverTest.php [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+    }
+
+    /**
+     * @return \Generator<string, array{0: string}>
+     */
+    public function provideDefaultConfigFileName(): \Generator
+    {
+        $defaultConfigFileNames = [
+            '.twig_cs',
+            '.twig_cs.dist',
```
Comment (localheinz) [inline tests/Config/ConfigResolverTest.php:280]: @Ciloe @OwlyCode @yguedidi

In the context of #247, we could add more data sets here to ensure that these default configuration files are resolved - what do you think?

File: tests/Config/ConfigResolverTest.php (hunk 1 above)
Comment (yguedidi) [reply tests/Config/ConfigResolverTest.php:152]: sure, I'll rework my PR when this is merged

**Comment by localheinz** [comment]
Thank you, @yguedidi!

**Comment by Ciloe** [comment]
Many thanks :)

**Pull request merged**